tests/RandomTest.php

```diff
@@ -21,12 +21,8 @@ public function q(...$args)
         return new Query(...$args);
     }
 
-    /**
-     * @doesNotPerformAssertions
-     */
     public function testMiscInsert()
```

Comment:
@georgehristov @DarkSide666 ok to remove this test? I do not see any value in it...